addons/website/models/res_config_settings.py

```diff
@@ -19,15 +19,11 @@ def _default_website(self):
     website_country_group_ids = fields.Many2many(related='website_id.country_group_ids', readonly=False)
     website_company_id = fields.Many2one(related='website_id.company_id', string='Website Company', readonly=False)
     website_logo = fields.Binary(related='website_id.logo', readonly=False)
-    language_ids = fields.Many2many(related='website_id.language_ids', relation='res.lang',
-        readonly=False)
```

Comment:
you don't like PEP8 that bad? ;-)

Reply:
Was the occasion to get rid of one PEP8 error :)